### DISCUSSION TITLE
networkXmas

### DISCUSSION THREAD

**Discussion by deusebio** [networkx/networkx#5229] [Ideas]
A few days ago, I decided to create just for fun a couple of utility functions for #networkx with some extra Christmas flavour in it! I have wrapped them in this gist

https://gist.github.com/deusebio/397610ac1f12dfd77ee6412987deb963

I was wondering whether it could be of interest to include it in the package just for fun and as an additional graph generation utility 

Any thought on this would be very welcome

**Comment by MridulS** [comment]
@deusebio just tried this out, this is fun! Thanks for the code.

We should just probably keep this here in the discussion as adding graph generation utilities (even fun ones like this) increases the future maintenance burden for the developers.

If anyone was wondering how it looks like :)
Added some colours too!
<img width="484" alt="Screenshot 2022-01-08 at 13 17 17" src="https://user-images.githubusercontent.com/5363860/148638895-df35ee2c-beb5-44db-a80d-909018f582dc.png">

**Reply by deusebio** [reply]
yes, sure! It makes total sense! 

Glad you liked it! I've actually been using it a bit to wish merry Christmas in a nerdy way lately... ;)
